src/Smalot/PdfParser/Element.php

```diff
@@ -98,24 +89,12 @@ public function getContent()
         return $this->value;
     }
 
-    /**
-     * @return string
-     */
-    public function __toString()
+    public function __toString(): string
     {
         return (string) ($this->value);
     }
 
-    /**
-     * @param string   $content
-     * @param Document $document
-     * @param int      $position
-     *
-     * @return array
-     *
-     * @throws \Exception
-     */
-    public static function parse($content, Document $document = null, &$position = 0)
+    public static function parse(string $content, ?Document $document = null, int &$position = 0)
```

Comment:
To be consistent with other return-type changes.

```suggestion
    public static function parse(string $content, ?Document $document = null, int &$position = 0): array
```

Reply:
The return type `array` cannot be added as the extended class `ElementString` does not return an array from that function.